File: Resoluti/Resoluti/Entities/Person.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain.Entities
{
    public class Person
    {
        [Key]
        public Guid Id { get; set; }
        public string Name { get; set; }
        public string Surname { get; set; }
        public DateTime BirthDate { get; set; }
        public string Email  { get; set; }
        public string CPF { get; set; }
        public string RG { get; set; }
        public IList<Address> ?Adresses { get; set; }
        public IList<Contact> ?Contacts { get; set; }
        public User User { get; set; }
        [ForeignKey(nameof(UserId))]
        public Guid UserId { get; set; }
        

    }
}

File: front-end/src/app/interfaces/Person.ts
export interface addPerson
{
    name: string,
    surname: string,
    birthDate: string,
    email: string,
    cpf: string,
    rg: string,
    addresses: address[],
    contacts: contat[],
    userId:string | null
  }


 

  export interface address{
    street: string,
    number: number,
    cep: string,
    complemento: string,
    city: string,
    state: string
  }


  export interface contat{
    name: string,
    contactNumber: string,
    type: string
  }